src/components/common/TechnologyLogos/styles.js

```diff
@@ -0,0 +1,33 @@
+export default theme => ({
+  root: {
+    marginTop: 40
+  },
+  brand: {
+    width: '50%',
+    display: 'block',
+    textAlign: 'center',
+    float: 'left',
```

Comment:
Use CSS flexbox to layout the list of items. Remember, the list should be created in the home page.